uportal-war/src/main/java/org/apereo/portal/rest/PagsRESTController.java

```diff
@@ -48,9 +48,6 @@
      Tried to use ResponseEntities but the ran into conflicts when using <mvc:annotation-driven/>
     */
 
-    /** Help other Java classes use this API well. */
-    public static final String URL_FORMAT_STRING = "/api/v4-3/pags/%s.json";
```

Comment:
Moved to another class.

There was a class I was trying to move that referenced this field.